packages/main/src/TabContainer.js

```diff
@@ -380,6 +380,25 @@ class TabContainer extends UI5Element {
 		}
 	}
 
+	_onHeaderMouseDown(event) {
```

Comment:
You would probably need SPACE and ENTER handling in addition to mouse actions, but this could be added on later stage as well